.scalafix.conf

```diff
@@ -5,7 +5,6 @@ rules = [
   LeakingImplicitClassVal
   NoAutoTupling
   NoValInForComprehension
-  OrganizeImports
```

Comment:
This rule fails without any patch suggestions only for kafkaConnector. 
Played with the configs https://github.com/liancheng/scalafix-organize-imports but in the end I don't think it's worth it due to bad feedback on failures.